rtv/utils.py

```diff
@@ -118,7 +118,7 @@ def load_config():
 
     return defaults
 
-def text_input(window):
+def text_input(window, show_cursor=False, insert_mode=True):
```

Comment:
I encountered a few weird edge cases when I resized the screen while the textbox was open. 
I think a solution to this may be to cancel out of the textbox (mimic the esc key being pressed) if the window is resized. This could be easily added to the ``validate()`` function inside of ``text_input()``. What do you think?